lib/beaker-pe/install/pe_utils.rb

```diff
@@ -1298,6 +1309,63 @@ def install_higgs( higgs_host = master )
           do_higgs_install higgs_host, options
         end
 
+        #Installs PE with a PE managed external postgres
+        def do_install_pe_with_pe_managed_external_postgres(hosts, opts, is_upgrade=false)
+          step "Setup tmp installer directory and pe.conf" do
+
+            prepare_hosts(hosts,opts)
+            register_feature_flags!(opts)
+            fetch_pe(hosts,opts)
+
+            [master, database, dashboard, pe_postgres].uniq.each do |host|
+              prepare_host_installer_options(host)
+
+              unless is_upgrade
+                setup_pe_conf(host, hosts, opts)
+              end
+            end
+          end
+
+          unless is_upgrade
+            step "Initial master install, expected to fail due to RBAC database not being initialized" do
+              begin
+                execute_installer_cmd(master, opts)
+              rescue => e
```

Comment:
What exception class is being raised here?  Catching everything can make it difficult to track down other unrelated issues which will now all die in the raise below, with their original information lost.

Reply:
This is just catching Beaker returning the unexpected 1 exit code. Since there are a lot of reasons for why that would be returned on install I add some logic below to try to minimize possible unrelated issues being caught.

Reply:
Yes, but what's the actual class raised?  If we can narrow to that, then this won't halt and bury some other exception bubbling up at this point.